docs/pages/_document.js

```diff
@@ -38,7 +38,6 @@ export default class MyDocument extends Document {
       <html lang="en">
         <Head>
           <Analytics.GoogleScript id="UA-107832480-3" />
-          <script src="/static/libs/prism/prism.js" />
```

Comment:
I'm not sure why this was used, hopefully there aren't any unforeseen consequences.

Reply:
Nevermind, this was used for installing language support. I've since limited language support to be more strict about when we need to update prism.